nsqd/protocol_v2_test.go

```diff
@@ -227,11 +229,10 @@ func TestClientTimeout(t *testing.T) {
 		default:
 			_, err := nsq.ReadResponse(conn)
 			if err != nil {
-				goto done
+				break loop
```

Comment:
I thought I saw `continue loop` and `break loop` elsewhere, must've been somewhere else. I'll revert and update.